src/assets/stylesheets/hub-create.scss

```diff
@@ -0,0 +1,221 @@
+.create-panel {
+  display: flex;
+  flex-direction: column;
+
+  @media (max-width: 768px) {
+    align-items: center;
+  }
+
+  &__header {
+    color: $grey-text;
+    font-size: 1.2em;
+    margin-bottom: 14px;
+    margin-left: 12px;
+    text-shadow: 0px 0px 2px rgba(32, 32, 32, 1.0);
+
+    @media (max-width: 768px) {
+      text-align: center;
+    }
+  }
+
+  &__form {
+    position: relative;
+    width: 700px;
+
+    @media (max-width: 768px) , (max-height: 715px) {
+      width: auto;
+      width: 500px;
+    }
+
+    @media (max-width: 520px) {
+      width: 300px;
+    }
+
+    &__name {
+      @extend %rounded-border;
+      @extend %default-font;
+
+      color: white;
+      font-size: 2em;
+      background: black;
+      padding: 25px;
+      padding-left: 90px;
+      padding-right: 90px;
+      width: 100%;
+      line-height: 40px;
+      z-index: 1;
+
+      &::selection {
+	background-color: #2F80ED;
+	color: white;
+      }
+
+      &::-moz-selection {
+	background-color: #2F80ED;
+	color: white;
+      }
+
+      @media (max-width: 768px) , (max-height: 715px) {
+	min-width: auto;
+	font-size: 1.5em;
+	width: 100%;
+	text-align: center;
+      }
+
+      @media (max-width: 520px) {
+	padding-left: 15px;
+	font-size: 1.2em;
+      }
+      
+      &--expanded {
+	padding-top: 365px;
+
+	@media (max-width: 768px) , (max-height: 715px) {
+	  padding-top: 300px;
+	}
+
+	@media (max-width: 520px) {
+	  padding-top: 190px;
+	}
+      }
+    }
+
+    &__left_button {
+      position: absolute;
+      height: 100%;
+      width: 90px;
+      height: 95px;
+      left: 4px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      cursor: pointer;
+      bottom: 0;
+
+      @media (max-width: 520px) {
+	display: none;
```

Comment:
Hmm, this hides the button on my S8. Is that intentional?

Reply:
yes unfortunately it ends up truncating the text field too much. this means when you create a room on your phone you can't pick the scene. (yes this is bad.) what i probably should do is expand the thing automatically via a viewport check at runtime. what do you think?

Reply:
actually I might be able to refactor everything to be CSS-based for expansion so maybe can use media queries.

Reply:
The randomize button is still hidden after you expand it. Maybe we could move the text input on to its own row and have the randomize and submit button below it.
Not needed for this PR though.

Reply:
I put in the hack for now so it expands on small screens. Def should fix later though.